docs/source/user/transformations/aggregation-mean.ipynb

```diff
@@ -108,16 +108,44 @@
        "0.5"
       ]
      },
-     "execution_count": 54,
+     "execution_count": 4,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "sb_mean_trans = make_sized_bounded_mean(size=10, bounds=(0., 10.))\n",
+    "sb_mean_trans = dp.t.make_mean(*input_space)\n",
     "sb_mean_trans([5.])"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can check that a dataset is a member of a domain by calling `.member`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "False"
```

Comment:
It's a little surprising that this returns `False`, might want to explain why (size).